packages/edit-site/src/components/page-patterns/grid-item.js

```diff
@@ -86,49 +66,9 @@ function GridItem( { categoryId, item, ...props } ) {
 		'is-inactive': isNonUserPattern,
 	} );
 
-	const deletePattern = async () => {
-		try {
-			await __experimentalDeleteReusableBlock( item.id );
-			createSuccessNotice(
-				sprintf(
-					// translators: %s: The pattern's title e.g. 'Call to action'.
-					__( '"%s" deleted.' ),
-					item.title
-				),
-				{ type: 'snackbar', id: 'edit-site-patterns-success' }
-			);
-		} catch ( error ) {
-			const errorMessage =
-				error.message && error.code !== 'unknown_error'
-					? error.message
-					: __( 'An error occurred while deleting the pattern.' );
-			createErrorNotice( errorMessage, {
-				type: 'snackbar',
-				id: 'edit-site-patterns-error',
-			} );
-		}
-	};
-	const deleteItem = () =>
-		isTemplatePart ? removeTemplate( item ) : deletePattern();
```

Comment:
TODO:

Still working out whether we still need `removeTemplate` for clearing customizations to theme templates/patterns.

Currently, this PR uses `revertTemplate`. Does it have the same effect?

Reply:
As far as I tested it, there was no problem, but I'm a little unsure. Does @aaronrobertshaw know more about this?

Reply:
My memory is a bit foggy on this one, so take this with a grain of salt.

As I understand it, there is a difference between `removeTemplate` and `revertTemplate` in that if the template doesn't have a theme file it won't be classed as revertable and an error notice is displayed with no further action taken.

If it's testing ok as Aki mentions then I'm probably missing something.